### PR TITLE
Ensured all objects have box colliders

### LINKED ISSUE #155: Check all objects for box colliders
## Expected Behaviour

Check all objects in the game have object colliders

## Actual Behaviour

Some objects can be walked through

## Steps to Reproduce Behaviour

1. See if you can walk through an object

### PR DESCRIPTION
**Pull request by Alice-Jaffray**
**Description of work.**

All objects now have box colliders

**To test:**

Attempt to go through all objects

Fixes #155

### CODE REVIEW

File: project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs [hunk 1]
```diff
@@ -12,6 +12,8 @@ public class MainMenuController : MonoBehaviour
     public GameObject customizeMenu;
     public GameObject creditsMenu;
 
+    public GameObject[] buttons;
+
```
Comment (cailafinn) [inline project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:16]: Not sure why this commit is making changes in these files?

File: project/sherLOCKED/Assets/Rooms/Misc/MainMenu.unity [hunk 2]
```diff
@@ -3766,6 +3766,12 @@ MonoBehaviour:
   levelSelectMenu: {fileID: 201798757}
   customizeMenu: {fileID: 676056322}
   creditsMenu: {fileID: 1406565016}
+  buttons:
+  - {fileID: 1343646979}
+  - {fileID: 1198231701}
+  - {fileID: 92819063}
+  - {fileID: 1951135330}
+  - {fileID: 846675922}
```
Comment (cailafinn) [inline project/sherLOCKED/Assets/Rooms/Misc/MainMenu.unity:3774]: These changes too

**Review by cailafinn — CHANGES_REQUESTED** [review]
The box Jason pointed out in Case 1 still has the same issue it did before. It may need changing size.

There's also some stuff changed in this commit that looks like it shouldn't be

File: project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs (hunk 1 above)
Comment (Alice-Jaffray) [reply project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:16]: neither do i - it happened during the reset ?

File: project/sherLOCKED/Assets/Rooms/Misc/MainMenu.unity (hunk 2 above)
Comment (Alice-Jaffray) [reply project/sherLOCKED/Assets/Rooms/Misc/MainMenu.unity:3774]: these ones too

File: project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs (hunk 1 above)
Comment (cailafinn) [reply project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:16]: Alright, see if you can reset them to what's on origin/master, we should try and avoid mixing commits too much

**Comment by Alice-Jaffray** [comment]
Created a different branch & pr for this ticket

**Pull request closed without merging**